third_party/terraform/website/docs/d/redis_instance.html.markdown

```diff
@@ -9,16 +9,26 @@ description: |-
 
 # google\_redis\_instance
 
-Get information about a Google Cloud Redis instance. For more information see
-the [official documentation](https://cloud.google.com/memorystore/docs/redis)
-and [API](https://cloud.google.com/memorystore/docs/redis/apis).
+Get info about a Google Cloud Redis instance from its name.
```

Comment:
```suggestion
Get info about a Google Cloud Redis instance.
```

No need to call out the required attributes imo (I didn't notice the example I linked did that)